content/docs/concepts/security/index.md

```diff
@@ -512,6 +512,7 @@ services in an Istio Mesh. It features:
 - **Flexibility through custom properties support**, for example conditions,
   in roles and role-bindings.
 - **High performance**, as Istio authorization is enforced natively on Envoy.
+- **Supports both HTTP and plain TCP protocol**.
```

Comment:
Also supports HTTPS, and HTTP2 natively.  I think this will render oddly compared to the rest of the text.  Some type of text should be after the trailing `**..` . I am not sure what that text should be.

Reply:
I rephrased it to `Supports HTTP, HTTPS and HTTP2 natively, as well as any plain TCP protocols`, does this look good to you?